src/index.py

```diff
@@ -42,6 +42,7 @@ def setup_config() -> FunctionConfig:
             runtime_key=os.getenv("INPUT_CDF_RUNTIME_CREDENTIALS"),
             cdf_base_url=os.getenv("INPUT_CDF_BASE_URL"),
         ),
+        secret=os.getenv("INPUT_FUNCTION_SECRETS") or None,
```

Comment:
why `or None`? `os.getenv("") `returns None by default

Reply:
if secret is an empty string, this would then return None

Reply:
Ah..

Reply:
is it necessary, no idea 😂

Reply:
I don't want to rely on default behavior of GitHub (aka they suddenly put empty strings)

Reply:
Why would github ever change any behaviour? :trollface: (master)

Reply:
Add a test for empty string scenario btw ;) :trollface: